projects/convex-hull/index.md

```diff
@@ -42,8 +42,18 @@ problem and the different algorithms to solve it.
 
 ## Testing
 
-Verify that the actual output matches the expected output. See the
-[requirements][3] section for an example of the expected output.
+The following table contains various test cases that you can use to
+verify the correctness of your solution; please note that different algorithms
+could produce the output starting from a different point, and/or in the opposite direction:
+
+| Description           | Input X | Input Y | Output |
+|-----------------------|---------|---------|--------|
+| X-Ordered triangle    | 100, 180, 240                         | 220, 120, 20                         | (100, 220)<br>(240, 20)<br>(180, 120)                            |
+| Pentagon, clockwise   | 100, 140, 320, 480, 280               | 240, 60, 40, 200, 300                | (100, 240)<br>(140, 60)<br>(320, 40)<br>(480, 200)<br>(280, 300) |
+| Cluster in center     | 260, 280, 300, 320, 600, 360, 20, 240 | 160, 100, 180, 140, 160, 320, 200, 0 | (20, 200)<br>(240, 0)<br>(600, 160)<br>(360, 320)                |
+| Missing y             | 100, 180, 240                         |                                      | Error returned                                                   |
```

Comment:
Can we specify the error string here? We usually do something like this: "Usage: please provide x and y coordinates as separate lists (e.g. "100, 440, 210")".

Reply:
Done, let me know if this is ok

Reply:
I think these are fine, but perhaps too specific. Keep in mind that this program has to be implemented in as many languages as possible, so reducing complexity is always good. Maybe stick to the generic error message from above.

Reply:
Ah, got it.. I based the error messages on the actual code I made; I will update it as well then